server.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const Auth = require('./auth.js');
+
+const app = express();
+app.use(bodyParser.json());
+
+const server = app.listen(process.env.PORT || 8080, () => {
+  const port = server.address().port;
+  console.log(`App now running on port ${port}`);
+});
+
+const auth = new Auth();
+
+app.post('/user/register', async (req, res) => {
+  const credentials = req.body;
```

Comment:
I'd go for `const { username, password } = req.body` but I'm a psychopath

Reply:
this is one of my favorite lines from these review comments

Reply:
Shit, then I don't know if I should tell you that I think that this would work:
```js
async ({ body: { username, password } }, res) => {
  console.log(username);
  console.log(password);
});
```